src/js/plugins/plugin.mobile_nav.js

```diff
@@ -70,9 +77,32 @@ BookReader.prototype.initToolbar = (function (super_) {
         }
       });
 
-      // High contrast button
-      $drawerEl.find('.high-contrast-button').click(
-        () => this.refs.$br.toggleClass('high-contrast'));
+
+      // Dinamically creates styles combining different filters for BookReaders imgs
+      // based on filters checkbox
+      $drawerEl.find('.BRcheckbox-filters').click(
+        () => {
```

Comment:
Let's pull this function out into it's function; maybe something like `updateFilters`? That way we can try to target it in unit tests.